src/App.css

```diff
@@ -0,0 +1,33 @@
+.App {
```

Comment:
I suggest you remove the contents in this file and leave it blank since it contains styling which we do not need.